src/lib/components/schedule/schedules-calendar-view.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+  import type { Schedule } from '$types';
+  import Tab from '$lib/holocene/tab.svelte';
+  import SchedulesIntervalView from './schedules-interval-view.svelte';
+  import ScheduleDayOfWeekView from './schedule-day-of-week-view.svelte';
+  import ScheduleDayOfMonthView from './schedule-day-of-month-view.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import { page } from '$app/stores';
+  import ScheduleFrequency from './schedule-frequency.svelte';
+
+  let scheduleId = $page.params.schedule;
+
+  let preset: SchedulePreset = scheduleId ? 'existing' : 'interval';
+
+  export let schedule: FullScheduleSpec | null = null;
+  export let daysOfWeek: string[];
+  export let daysOfMonth: number[];
+  export let months: string[];
+  export let days: string;
+  export let hour: string;
+  export let minute: string;
+  export let second: string;
+  export let phase: string;
+  export let cronString: string;
+
+  const clearSchedule = (_preset: SchedulePreset) => {
```

Comment:
nit: looks like `_preset` is unused.

Reply:
I'm using preset to trigger the function in $: {}, but I don't need to use it. Thus the _preset. Kind of a weird pattern but I don't mind it.

Reply:
Ohh interesting. That makes sense 👍